src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/alert-infrastructure-processes.mdx

```diff
@@ -86,9 +86,19 @@ By applying filters to the hosts and processes that are important to your busine
 
 To define the **Process running** alert criteria:
 
-1. Follow standard procedures to [create an infrastructure alert condition](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-alerts-add-edit-or-view-host-alert-information).
-2. Select **Process running** as the **Alert type**.
-3. Filter what hosts and processes you want the alert condition to apply to.
-4. Define the **Critical** threshold for triggering the alert notification: minimum 1 minute, default 5 minutes, maximum 60 minutes.
+1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Infrastructure**. Mouse over a chart you want to alert on, select the ellipses <Icon name="fe-more-horizontal"/> icon, and then select **Create alert condition**.
+2. Type a meaningful condition name.
+3. Select **Process running** as the **Alert type**.
+4. Filter what hosts and processes you want the alert condition to apply to.
+5. Define the **Critical** threshold for triggering the alert notification: minimum 1 minute, default 5 minutes, maximum 60 minutes.
+6. Optional: To create the condition criteria proactively but not receive alert notifications at this time, turn off the [**Enabled** checkbox option](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-alerts-add-edit-or-view-host-alert-information#enabled).
```

Comment:
There might be a clearer way to state this. I'm not quite sure what this is saying... Are you saying that you can create alert condition but disable notifications for it?

Reply:
@paperclypse I just brought over the steps from the infra doc, so I can remove these optional steps. I'll check w/ Sammie too.